.github/workflows/ansible-test.yml

```diff
@@ -63,7 +63,15 @@ jobs:
           - stable-2.14
           - devel
         # - milestone
-    runs-on: ubuntu-latest
+    # Ansible-test on various stable branches does not yet work well with cgroups v2.
+    # Since ubuntu-latest now uses Ubuntu 22.04, we need to fall back to the ubuntu-20.04
+    # image for these stable branches. The list of branches where this is necessary will
+    # shrink over time, check out https://github.com/ansible-collections/news-for-maintainers/issues/28
+    # for the latest list.
+    runs-on: >-
+      ${{ contains(fromJson(
+          '["stable-2.9", "stable-2.10", "stable-2.11", "stable-2.12", "stable-2.13", "stable-2.14", "milestone"]'
+      ), matrix.ansible) && 'ubuntu-20.04' || 'ubuntu-latest' }}
```

Comment:
what about "devel"?

Reply:
`devel` has https://github.com/ansible/ansible/pull/78550 itself in it, it does not need a backport of that PR :)